README.md

```diff
@@ -139,27 +139,27 @@ exclude-labels:
 
 Pull requests with the label "skip-changelog" will now be excluded from the release draft.
 
-## Projects that don't use Semantic Versioning
+## Projects that don’t use Semantic Versioning
 
-If your project doesn't follow [Semantic Versioning](https://semver.org) you can still use Release Drafter, but you may want to set the `version-template` option to customize how the `$NEXT_{PATCH,MINOR,MAJOR}_VERSION` environment variables are generated.
+If your project doesn’t follow [Semantic Versioning](https://semver.org) you can still use Release Drafter, but you may want to set the `version-template` option to customize how the `$NEXT_{PATCH,MINOR,MAJOR}_VERSION` environment variables are generated.
 
-For example, if your project doesn't use patch version numbers, you can set `version-template` to `$MAJOR.$MINOR`. If the current release is version 1.0, then `$NEXT_MINOR_VERSION` will be `1.1`.
+For example, if your project doesn’t use patch version numbers, you can set `version-template` to `$MAJOR.$MINOR`. If the current release is version 1.0, then `$NEXT_MINOR_VERSION` will be `1.1`.
 
 ## Replacers
 
-You can search and replace content in the generated changelog body using the `replacers` option. Replacers support searching via regular expressions with the `regex` option, and plain strings with the `search` option. Each replacer is applied in order.
+You can search and replace content in the generated changelog body using the `replacers` option. Replacers support searching via regular expressions. Each replacer is applied in order.
 
 ```yml
 replacers:
-  - regex: '/CVE-(\d{4})-(\d+)/g'
+  - search: '/CVE-(\d{4})-(\d+)/g'
```

Comment:
What’s the story with changing this again?

Reply:
I voiced my opinion in the #185 partly based on what I already knew would happen in the schema validation.

If you have a look at the code in #185 and my comment https://github.com/toolmantim/release-drafter/pull/185/files#r279162469

This is the very reason for keeping with the name `search`.
When the developer also referred to it as `search` inside the code.

As I said it increases the complexity for both users and developers.
Right now we have a simple function that can handle the use case for when a string is in regex representation and when it is a simple string.

With the regex kept in the schema, I would have to do something like this.
```
   customReplacers.forEach(({ search, regex, replace }) => {
      if (regex) search = regex
      str = str.replace(search, replace)
    })
```
Or add to the schema logic by using `xor` and/or `rename`

This is what had me confused in #185 changing the schema is not OK.
https://github.com/toolmantim/release-drafter/blob/5b573591f9055ce2280b41f5fb1b35beb6350201/lib/template.js#L34-L45

Reply:
I'm confused only because I thought changing it to be explicit (aka having two different config keys) was your idea? https://github.com/toolmantim/release-drafter/pull/185#discussion_r278704617 😁

Reply:
I think there might have been a misunderstanding, in that both @halkeye and myself assumed you wanted an explicit parameter (and I just trusted you both had talked through the design enough) 😁

Reply:
My earlier comment was against adding another property: 

https://github.com/toolmantim/release-drafter/pull/185#discussion_r278671766

> I'd prefer not adding another property and instead suggest you look at [regex-parser](https://www.npmjs.com/package/regex-parser)
> 
> Should handle parsing the regex in both formats only concern is to take the regex you get and add the default `g` flag

Reply:
Ah I'm sorry, I don't think that came across very clearly to me. I don't think myself or @halkeye cared too much, and actually thought this was what you wanted. It's not a big deal to change it!

Want to open a separate PR just to change it to use `search`? And I'll update the readme too (I tried my best to simplify/explain the separate `regex`/`search` sub-properties.

Reply:
Feel free to open a PR. I will rebase my changes on top.

Reply:
The readme in this PR seems fine.

Reply:
Not quite, it reads a bit strangely as it was structured for the old syntax. I’ve just added a suggested edit though 👍🏼